frontend/src/stimulus/controllers/dynamic/recurring-meetings/form.controller.ts

```diff
@@ -25,7 +25,10 @@ export default class OpRecurringMeetingsFormController extends ApplicationContro
       .request(
         `${this.pathHelper.staticBase}/recurring_meetings/humanize_schedule?${urlSearchParams.toString()}`,
         {
-          headers: { Accept: 'text/vnd.turbo-stream.html' },
+          headers: {
+            Accept: 'text/vnd.turbo-stream.html',
+            'X-Authentication-Scheme': 'Session',
```

Comment:
Why add it here multiple times instead of adding it to TurboRequestsService#request directly? I don't think this is ever going to be used outside session authentication